easybuild/easyconfigs/o/OpenSlide/276.diff

```diff
@@ -0,0 +1,213 @@
+diff --git a/src/openslide-decode-tifflike.c b/src/openslide-decode-tifflike.c
```

Comment:
Would be good to have the description and link in the patch itself instead.

I'm also not sure crazy about the name. What about `OpenSlide-3.4.1_large_file_support.patch`